### ISSUE TITLE
Warn user if they are applying a style to a view that was never: build, create, or append

### ISSUE THREAD

**Issue by twerth** [infinitered/rmq#263]

**Comment by squidpunch** [comment]
@twerth do you mean `apply_style(:blah)` should show a warning?  Or something else?  I could spend some time implementing this, but I want to be sure what's the specification!

**Comment by twerth** [comment]
yes, if the view isn't `built`, meaning `build`, `create`, or `append`. Because if it's not, it may not have the correct stylesheet, etc. This causes problems for some people.

**Comment by GantMan** [comment]
:+1:

**Comment by squidpunch** [comment]
hrmm interesting, I often build with a style, then based on some activity on the view apply a new style (which wouldnt be built).  Wonder if its worth allowing a setting to quiet this warning if you dont want to see it - because I could see having this warning pop up a few places in my app because of how I am currently using it (but correctly....or at least in my opinion its correctly...)
